mlflow/server/js/src/experiment-tracking/components/ExperimentRunsTableMultiColumnView2.js

```diff
@@ -673,27 +693,32 @@ function SourceCellRenderer(props) {
   );
 }
 
-SourceCellRenderer.propTypes = { data: PropTypes.object };
+SourceCellRenderer.propTypes = { value: PropTypes.object };
 
 function VersionCellRenderer(props) {
-  const { tags } = props.data;
-  return Utils.renderVersion(tags) || EMPTY_CELL_PLACEHOLDER;
+  const { version, name, type } = props.value;
+  return Utils.renderSourceVersion(version, name, type) || EMPTY_CELL_PLACEHOLDER;
 }
 
-ExperimentNameRenderer.propTypes = { data: PropTypes.object };
+VersionCellRenderer.propTypes = { value: PropTypes.object };
+
 function ExperimentNameRenderer(props) {
-  const { experimentId, experimentName, experimentBasename } = props.data;
+  const { experimentId } = props.data;
+  const { name, basename } = props.value;
```

Comment:
What's the difference between `data` and `value` here? ==> The question behind this question is really, why are we reading `experimentId` in `DateCellRenderer` off of `value` there, but off of `data` here?

Reply:
`data` contains the data for the whole row, whereas `value` is the value for this cell (selected from the row data using the `field` configured for the column). The `value` here doesn't include the `experimentId` because I'm using the experiment name values directly from the map returned by `Utils.getExperimentNameMap` in `getRowData`.

I could probably create new value objects that also include the `experimentId` which might be tidier, but that seems unnecessary as the experiment id for a row is never going to change (if it could change it would be important to include it so that the equality comparison was correct).

Reply:
I've added a comment in the function to explain this too.